api/api-base/src/main/java/com/thoughtworks/go/api/representers/configorigin/ConfigRepoOriginRepresenter.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.thoughtworks.go.apiv9.admin.shared.representers.configorigin;
+package com.thoughtworks.go.api.representers.configorigin;
```

Comment:
I don't think we should be sharing representers between API versions. It's ok for each version to have it's own set of shared representers.

Reply:
IMO, there are certain entities for which we have a fixed representation for all APIs. E.g. environment variable or plugin configuration property. Describing origin is also one such thing.
Hence, moving the same to api-base is a better option.

Reply:
I understand they have a same representation hence they are in a common place. We keep bumping up versions often and also delete the older versions post deprecation. I think getting into a mode of using representers across multiple versions would lead to confusion and cleaning up the representers would be hard as well.

Reply:
Done.